ncm-metaconfig/src/main/metaconfig/icinga-web/pan/schema.pan

```diff
@@ -0,0 +1,41 @@
+declaration template metaconfig/icinga-web/schema;
+
+include 'pan/types';
+
+type iw_caching_opts = {
+    'enabled' : boolean
+    'driver' : string with match (SELF, "^(apc|memcache)$")
+    'use_query_cache' : boolean
+    'use_result_cache' ? boolean
+    'result_cache_lifespan' ? long(0..) = 60
+};
+
+type icinga_database_dsn = {
+    'protocol' : string with match (SELF, "^(pgsql|mysql|oracle)$")
```

Comment:
Yes, I read that but for icingaOracle it can get very complex if you don't use standard dsn. Even if I could make sense of all these oracle specifics (there really is no good documentation from icinga for this) I have no means to test it. This PR was made to cover most standard/basic options.